clients/cobol-lsp-vscode-extension/src/test/suite/lsp.spec.test.ts

```diff
@@ -55,7 +56,7 @@ suite('Integration Test Suite', () => {
 
     test('TC152080 Find all references from the word middle', async () => {
         const result: vscode.Location[] = await vscode.commands.executeCommand('vscode.executeReferenceProvider', editor.document.uri, new vscode.Position(20, 15));
-        await helper.sleep(1000);
+        await helper.sleep(2000);
```

Comment:
Any specific reason for increasing the sleep time?

Reply:
Yes. To make them pass

Reply:
Thanks, Niharika!
Updated the test cases.